python/dlisio/plumbing/channel.py

```diff
@@ -130,18 +130,18 @@ def source(self):
 
     @property
     def index(self):
```

Comment:
Btw, I agree with Jørgen that this property shouldn't be present for channels and should be accessed through frame.